src/Shouldly/Shouldly.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Shouldly - Assertion framework for .NET. The way asserting *Should* be</Description>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>net451;net452;netstandard1.3</TargetFrameworks>
```

Comment:
Is there a particular reason you need to support net451 and net452? there's virtually no API surface area different between them; in general, I suggest targeting the fewest frameworks you need to support.